### DISCUSSION TITLE
can't display search result on hyprland

### DISCUSSION THREAD

**Discussion by WhateverMO** [Ulauncher/Ulauncher#1218] [Q&A]
using ulauncher --no-window-shadow
![image](https://user-images.githubusercontent.com/75033001/230724653-80195c0a-296e-46b0-8167-2e059e105e0f.png)
using ulauncher
![image](https://user-images.githubusercontent.com/75033001/230724757-c7b51c45-51d0-48ad-8b8c-d0a53b1da845.png)

**Comment by friday** [comment]
Consult the hyprland documentation or devs/repo how to avoid this.
